### PR TITLE
#BD-2194 Commenting on email templates

### PR DESCRIPTION
**Pull request by rachel-feinberg**
# Pull Request/Issue Resolution

Closes #**BD-2194**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that you haven't removed any images (replacing an image with an updated one of the same name is fine), as this breaks the French site
- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @josh-mccrowell-braze and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/email/templates/email_template.md [hunk 1]
```diff
@@ -170,6 +170,9 @@ To use your email for an API campaign, you need an `email_template_id`, which ca
 
 You can [duplicate]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/duplicate/) and [archive]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/archive/) email templates! Learn more about creating and managing templates and creative content in [Templates & Media]({{site.baseurl}}/user_guide/engagement_tools/templates_and_media/).
 
+## Commenting on email templates
+You can comment on email templates in the drag-and-drop editor. Click the Content Block in the email body that you’d like to comment on, then select the <i class="fas fa-comment"></i> comment icon. This opens a sidebar where you can enter your comment through the text input field by clicking **Submit**. Click **Done**, then **Save Template** to save your comment and make it visible to other users.
+
```
Comment (KellieHawks) [inline _docs/_user_guide/message_building_by_channel/email/templates/email_template.md:175]: Hey @rachel-feinberg, I think you could also add something like:

Once saved, users will see floating <insert-comment-icon-here> icons over unaddressed Content Block comments. Selecting *Resolve** will resolve these comments. 

Insert small picture example"

**Review by KellieHawks — CHANGES_REQUESTED** [review]
Minor changes requested. TY!

File: _docs/_user_guide/message_building_by_channel/email/templates/email_template.md (hunk 1 above)
Comment (lydia-xie) [reply _docs/_user_guide/message_building_by_channel/email/templates/email_template.md:175]: +1 - an image here of what comments look like could be helpful!

Comment (rachel-feinberg) [reply _docs/_user_guide/message_building_by_channel/email/templates/email_template.md:179]: @KellieHawks @lydia-xie I made a few edits. I had a hiccup with adding the screenshot, but I think I fixed it!

**Review by KellieHawks — APPROVED** [review]

**Comment by KellieHawks** [comment]
Looks great! Congrats on your first PR!

**Pull request merged**